src/main/resources/assets/api-docs.yaml

```diff
@@ -4125,6 +4123,9 @@ components:
         objectId:
           type: string
           description: The sample collection ID, if defined
+        dataUse:
+          type: object
```

Comment:
Outside the scope of this PR - it would be nice to have a model for `DataUse`

Reply:
Filed a ticket for this here: https://broadinstitute.atlassian.net/browse/DUOS-855